### PR TITLE
Improve no IPv6 ranges in subnet error message

### PR DESCRIPTION
**Pull request by panslava**
/assign cezarygerard

Cynthia suggested better error message, and I agree

### CODE REVIEW

**Comment by panslava** [comment]
/assign @aojea 

could you please take a look

**Comment by panslava** [comment]
/unassign aojea
/assign @code-elinka 

or, I think Antonio is out, maybe Elina can help to merge that

File: pkg/loadbalancers/l4ipv6.go [hunk 1]
```diff
@@ -208,7 +208,9 @@ func (l4 *L4) serviceSubnetHasInternalIPv6Range() error {
 	if !hasIPv6SubnetRange {
 		// We don't need to emit custom event, because errors are already emitted to the user as events.
 		klog.Infof("Subnet %s for IPv6 Service %s/%s does not have internal IPv6 ranges", subnetName, l4.Service.Namespace, l4.Service.Name)
-		return utils.NewUserError(fmt.Errorf("subnet %s does not have internal IPv6 ranges, required for IPv6 Service", subnetName))
+		return utils.NewUserError(
+			fmt.Errorf(
+				"subnet %s does not have internal IPv6 ranges, required for an internal IPv6 Service. You can specify an internal IPv6 subnet using the \"networking.gke.io/load-balancer-subnet\" annotation on the Service", subnetName))
```
Comment (code-elinka) [inline pkg/loadbalancers/l4ipv6.go:213]: nit: add annotation key instead of string itself

Comment (panslava) [reply pkg/loadbalancers/l4ipv6.go:213]: oh, definitely it will be better, changed

**Comment by panslava** [comment]
aaand lgtm disappeared after small change :D

@code-elinka could you please take a look again

**Comment by panslava** [comment]
some strange error "starting e2e cluster: error" 

/retest

**Comment by aojea** [comment]
> @panslava: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> 
> Test name	Commit	Details	Required	Rerun command
> pull-ingress-gce-e2e	[9412d41](https://github.com/kubernetes/ingress-gce/commit/9412d414a872afd703b5f8e41d24abb35fd6e256)	[link](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/ingress-gce/2196/pull-ingress-gce-e2e/1685036373431554048)	true	`/test pull-ingress-gce-e2e`
> [Full PR test history](https://prow.k8s.io/pr-history?org=kubernetes&repo=ingress-gce&pr=2196). [Your PR dashboard](https://prow.k8s.io/pr?query=is%3Apr%20state%3Aopen%20author%3Apanslava). Please help us cut down on flakes by [linking to](https://git.k8s.io/community/contributors/devel/sig-testing/flaky-tests.md#filing-issues-for-flaky-tests) an [open issue](https://github.com/kubernetes/ingress-gce/issues?q=is:issue+is:open) when you hit one in your PR.

The job is failing because of  https://cdn.dl.k8s.io/gci-mounter/mounter.tar

> Jul 28 21:27:29.132278 e2e-afb2bf35b8-7a98c-master configure.sh[1322]: Warning: Problem : HTTP error. Will retry in 10 seconds. 1 retries left.
Jul 28 21:27:39.270759 e2e-afb2bf35b8-7a98c-master configure.sh[1322]: curl: (22) The requested URL returned error: 503
Jul 28 21:27:39.273030 e2e-afb2bf35b8-7a98c-master configure.sh[1092]: == Failed to download https://dl.k8s.io/gci-mounter/mounter.tar. Retrying. ==
Jul 28 21:27:39.510777 e2e-afb2bf35b8-7a98c-master configure.sh[1324]: curl: (22) The requested URL returned error: 503

/cc @ameukam @BenTheElder

**Comment by aojea** [comment]
> > @panslava: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> > Test name	Commit	Details	Required	Rerun command
> > pull-ingress-gce-e2e	[9412d41](https://github.com/kubernetes/ingress-gce/commit/9412d414a872afd703b5f8e41d24abb35fd6e256)	[link](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/ingress-gce/2196/pull-ingress-gce-e2e/1685036373431554048)	true	`/test pull-ingress-gce-e2e`
> > [Full PR test history](https://prow.k8s.io/pr-history?org=kubernetes&repo=ingress-gce&pr=2196). [Your PR dashboard](https://prow.k8s.io/pr?query=is%3Apr%20state%3Aopen%20author%3Apanslava). Please help us cut down on flakes by [linking to](https://git.k8s.io/community/contributors/devel/sig-testing/flaky-tests.md#filing-issues-for-flaky-tests) an [open issue](https://github.com/kubernetes/ingress-gce/issues?q=is:issue+is:open) when you hit one in your PR.
> 
> The job is failing because of https://cdn.dl.k8s.io/gci-mounter/mounter.tar
> 
> > Jul 28 21:27:29.132278 e2e-afb2bf35b8-7a98c-master configure.sh[1322]: Warning: Problem : HTTP error. Will retry in 10 seconds. 1 retries left.
> > Jul 28 21:27:39.270759 e2e-afb2bf35b8-7a98c-master configure.sh[1322]: curl: (22) The requested URL returned error: 503
> > Jul 28 21:27:39.273030 e2e-afb2bf35b8-7a98c-master configure.sh[1092]: == Failed to download https://dl.k8s.io/gci-mounter/mounter.tar. Retrying. ==
> > Jul 28 21:27:39.510777 e2e-afb2bf35b8-7a98c-master configure.sh[1324]: curl: (22) The requested URL returned error: 503
> 
> /cc @ameukam @BenTheElder

it was fixed it should be good now

**Pull request merged**